LoadingUI/LoadingUI/DrawingController.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  DrawingController.swift
+//  LoadingUI
+//
+//  Created by Cora Jacobson on 10/6/20.
+//
+
+import Foundation
+import SceneKit
+
+internal class DrawingController {
+    private let cupColor = UIColor(red: 0/255, green: 200/255, blue: 200/255, alpha: 0.75)
+    private let teaColor = UIColor(red: 180/255, green: 120/255, blue: 50/255, alpha: 1)
+    internal let tea = SCNCylinder(radius: 1.4, height: 0)
+    internal var teaNode = SCNNode()
```

Comment:
Just fyi, if you don't say "internal" explicitly, everything is internal by default.